pytorch_lightning/utilities/distributed.py

```diff
@@ -203,6 +203,7 @@ def all_gather_ddp_if_available(
     Return:
         A tensor of shape (world_size, batch, ...)
     """
+    group = group if group is not None else torch.distributed.group.WORLD
```

Comment:
Should this be moved inside the `if` statement as `torch.distributed.group.WORLD` is not defined on windows?